docs/content/concepts/graphql-rpc.mdx

```diff
@@ -202,7 +202,7 @@ Fields that return a paginated response accept at least the following optional p
 
 They also return a type that conforms to the [GraphQL Cursor Connections Specification](https://relay.dev/graphql/connections.htm), meaning its name ends in `Connection`, and it contains at least the following fields:
 
-- `pageInfo`, of type [PageInfo](../references/sui-api/sui-graphql/reference/objects/page-info.mdx), which indicates whether there are more pages before or after the page returned.
+- `pageInfo`, of type <a href="/references/sui-api/sui-graphql/reference/objects/page-info" data-noBrokenLinkCheck='true'>PageInfo</a>, which indicates whether there are more pages before or after the page returned.
```

Comment:
Out of curiosity, this is now the recommended way to point to a page inside our docs?

Reply:
No. It's the way we must point to docs now that are automatically generated.